refrepo_git.py

```diff
@@ -165,6 +167,8 @@ def extract_remote_conf(remotes):
         parts = remote.split()
         if parts[2] == "(fetch)":
             url = parts[1]
+            if os.getenv(_DROP_CREDENTIALS_ENV_VAR):
```

Comment:
I think that the string needs to be converted to a boolean, right? This will only check if the environment variable is non-empty, which is not the same as "not falsy", IIUC.

Reply:
Ok, how about this? 

```python
if os.getenv(_DROP_CREDENTIALS_ENV_VAR).lower() in ('true', '1', 't', 'yes', 'y'):
```

Reply:
Should work. Add "on" too. And put the values in `_TRUTHY_VALUES`.

Reply:
Done!